packages/@aws-cdk-testing/framework-integ/test/aws-rds/test/integ.cluster-multiaz.ts

```diff
@@ -0,0 +1,39 @@
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as kms from 'aws-cdk-lib/aws-kms';
+import * as cdk from 'aws-cdk-lib';
+import { Credentials, MultiAZDatabaseCluster, DatabaseClusterEngine, ParameterGroup } from 'aws-cdk-lib/aws-rds';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-rds-multiaz-integ');
+
+const vpc = new ec2.Vpc(stack, 'VPC', { maxAzs: 3, restrictDefaultSecurityGroup: false });
```

Comment:
With the current way that that integration test harness works this VPC will only ever have 2 AZ's which results in the `MultiAZDatabaseCluster` failing to deploy.